src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +160,33 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
+                if(allMethods.Count() > 0)
+                {
+                    var replaceMethod = allMethods.Where(m => m.Identifier.ToString() == methodName).FirstOrDefault();
+                    if(replaceMethod != null )
+                    {
+                        string[] modifierList = modifiers.Split(new char[] { ' ', ','});
+                        var verifiedModifiers = modifierList.Where(m => CTA.Rules.Config.Constants.supportedMethodModifiers.Contains(m.ToLower()));
+                        if(modifierList.Count() == verifiedModifiers.Count())
```

Comment:
Is it safe to use `.ToLower()` here? Modifiers are case-sensitive so this might allow something like `INTERNAL MyMethod(...)`

Also a suggestion that is open for debate: Instead of comparing the counts to validate, we can replace `.Where(...)` with `All(...)`. It will return a boolean stating whether or not all items in the collection match the condition. Up to you if you think this will improve readability. To add to this, validation can be placed in a separate method if you believe it will improve readability and/or if the validation logic is likely to be reused.